File: mario.c
//A program that prints out pyramids out of #hashes, takes a height of the pyramid as an input

#include <stdio.h>
#include <cs50.h>

int get_suitable_int(void);

int main(void)
{

    int height = get_suitable_int();

    for (int i = 1; i < height + 1; i++)
    {
        for (int s = 1; s < ((height + 1) - i); s++)
        {
            printf(" ");
        }

        
        for (int w = 1; w < (i + 1); w++)
        {
            printf("#");
        }
    
        printf("  ");
        
        for (int d = 1; d < (i + 1); d++)
        {
            printf("#");
        }
        
        printf("\n");
    }



}

int get_suitable_int(void)
{
    int n;
    do
    {
        n = get_int("suitable integer:");
    }
    while ( ! (n > 0 && n < 9));
    return n;

}

File: mario.py
# Takes as an input a number (a height) between 1 and 8
# Prints a pyramid made out of hash characters of the specified height
from cs50 import get_int


height = 0

# Get a suitable number as an input
while height < 1 or height > 8: :
    height = get_int("Height:")
    
    
# Iterate
for i in range(1, height + 1):

    # Hashes of the left-side pyramid
    print(" " * (height - i), end="")
    print("#" * i, end="")
    
    # Space between both pyramids
    print("  ",end="")
    
    # Hashes of the right-side pyramid
    print("#" * i, end="")
    
    # New line
    print()
